cirq/circuits/circuit_test.py

```diff
@@ -2688,60 +2688,64 @@ def test_submoments():
     )
 
     cirq.testing.assert_has_diagram(circuit, """
-         ┌───────────┐
-0: ───H───@─────────────@────────
-          │             │
-1: ───@───┼@────────────┼────────
-      │   ││            │
-2: ───@───┼┼────@───────┼────H───
-          ││    │       │
-3: ───H───@┼────┼───────X^0.5────
-           │    │
-4: ────────X^0.5┼───────H────────
-                │
-5: ─────────────X^0.5────────────
-         └───────────┘
+          ┌───────────┐
+0: ───H────@──────────────@────────
+           │              │
+1: ───@────┼@─────────────┼────────
+      │    ││             │
+2: ───@────┼┼────@────────┼────H───
+           ││    │        │
+3: ───H────@┼────┼────────X^0.5────
+            │    │
+4: ─────────X^0.5┼────────H────────
+                 │
+5: ──────────────X^0.5─────────────
+          └───────────┘
 """)
 
     cirq.testing.assert_has_diagram(circuit, """
   0 1 2 3     4     5
   │ │ │ │     │     │
   H @─@ H     │     │
-┌ │ │ │ │     │     │     ┐
+  │ │ │ │     │     │
+┌╴│ │ │ │     │     │    ╶┐
```

Comment:
Ah man, it doesn't render perfectly in chrome on linux

![image](https://user-images.githubusercontent.com/79941/52155354-2528d300-2637-11e9-8379-515b434e95d1.png)

Reply:
it does look good on mac / chrome